### ISSUE TITLE
int13fakev86io=true slows down boot

### ISSUE THREAD

**Issue by valnar1** [joncampbell123/dosbox-x#3495]
### Question

I'm following the recommended config file for a Windows 98 image and noticed that _int13fakev86io=true_ slows down the boot considerably.  If I remove that, all is well.

Why is this needed or recommended for a Win98 image?

I do recognize that without it, the IDE controller in device manager isn't present, however the image is much much faster.  Is there a downside?

### Have you checked that no similar question(s) exist?

- [X] I have searched and didn't find any similar question.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by rderooy** [comment]
I think you also mentioned this on Discord. As I replied there, the wiki has that setting as it is recommended for Win95 and later by @joncampbell123 

Have a look at the reference config and `src/hardware/ide.cpp` for more information.

**Comment by valnar1** [comment]
Yep that was me.  Perhaps a better question is, why is it so slow?  Does anyone else see that?

**Comment by joncampbell123** [comment]
It's perhaps the logging calls to show ATA commands are in use.
I've made a commit that disables that logging.

**Comment by joncampbell123** [comment]
Are you able to compile the latest commit?

**Comment by valnar1** [comment]
I cannot.  I will wait for the next release.

**Comment by valnar1** [comment]
The new release doesn't fix the slowness.
